ProductionModePlugin.js

```diff
@@ -67,34 +67,37 @@ class ProductionModePlugin {
         if(param.isString() && param.string === "globalize" && this.moduleFilter(request) &&
           !(globalizeCompilerHelper.isCompiledDataModule(request))) {
 
-          // Statically extract Globalize formatters and parsers from the request
-          // file only. Then, create a custom precompiled formatters/parsers module
-          // that will be called instead of Globalize, which in turn requires
-          // Globalize, set the default locale and then exports the Globalize
-          // object.
-          const compiledDataFilepath = globalizeCompilerHelper.createCompiledDataModule(request);
+          this.supportedLocales.forEach((locale) => {
```

Comment:
I would sadly say no: we need to create dependencies (one per locale) between the "real" js file and i18n's. That's a module dependency. I checked, they are all filepath based.

I'll do another pass though.

I had something else in mind for later: In theory, we don't need 1 dependency per locale per Globalize'd source file, nor 1 per file like today.

We need 1 dependency (`module.exports = require(\"globalize\");` is enough) per locale, wiring the development locale one to all the "`Globalize = require…`.

We could use this pass to collect file changes information and only rebuild locales if the sources or the source i18n have changed (compilation takes time).